gantt_edit.php

```diff
@@ -62,7 +68,8 @@
 		}
 		if ( in_array( '', $infoLabel, true ) )
 		{
-			exit;
+			$validationMsg = 'At least one label is incomplete.';
```

Comment:
shouldn't we be appending to the validationMsg variable rather than re-setting it?  Also can one include any info on which label is incomplete, and if not, perhaps include "last complete label=xxxx" to help users locate the problem.

Reply:
same for other instances of setting this variable, so I won't repeat the comment.

Reply:
I think the configuration is simple enough that any errors are likely to be easy to identify, but this is an improvement which may be considered for a future version.